spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/kafka/KafkaPropertiesTests.java

```diff
@@ -128,42 +114,6 @@ void sslPropertiesWhenTrustStoreLocationAndCertificatesSetShouldThrowException()
 			.isThrownBy(properties::buildConsumerProperties);
 	}
 
-	@Test
-	void sslPropertiesWhenKeyStoreLocationAndBundleSetShouldThrowException() {
-		KafkaProperties properties = new KafkaProperties();
-		properties.getSsl().setBundle("myBundle");
-		properties.getSsl().setKeyStoreLocation(new ClassPathResource("ksLoc"));
-		assertThatExceptionOfType(MutuallyExclusiveConfigurationPropertiesException.class).isThrownBy(
-				() -> properties.buildConsumerProperties(new DefaultSslBundleRegistry("myBundle", this.sslBundle)));
-	}
-
-	@Test
-	void sslPropertiesWhenKeyStoreKeyAndBundleSetShouldThrowException() {
-		KafkaProperties properties = new KafkaProperties();
-		properties.getSsl().setBundle("myBundle");
-		properties.getSsl().setKeyStoreKey("-----BEGIN");
-		assertThatExceptionOfType(MutuallyExclusiveConfigurationPropertiesException.class).isThrownBy(
-				() -> properties.buildConsumerProperties(new DefaultSslBundleRegistry("myBundle", this.sslBundle)));
-	}
-
-	@Test
-	void sslPropertiesWhenTrustStoreLocationAndBundleSetShouldThrowException() {
```

Comment:
This test should be kept but should call `.isThrownBy(properties::buildConsumerProperties);`.